hashtagsv2/hashtags/templates/hashtags/all_users.html

```diff
@@ -0,0 +1,77 @@
+{% extends "hashtags/base.html" %}
+{% load static %}
+{% load url_replace %}
+
+{% block content %}
+<div class="container">
+    <div class="info">
+        <div class="row">
+            <div class="two-thirds column">
+            <h2>
+                {% for query in hashtag_query_list %}
+                #{{ query }}
+                {% endfor %}
+            </h2>
+            <h2>Users</h2>
+            </div>
+            <div class="one-third column">
+                <p class="stats-date-range">{{ oldest }} - {{ newest }}</p>
+                <table class="stats-table">
+                    <tr>
+                        <td class="stat">{{ revisions }}</td>
+                        <td class="stat-label">revision{{ revisions|pluralize:"s" }}</td>
+                    </tr>
+                    <tr>
+                        <td class="stat">{{ pages }}</td> 
+                        <td class="stat-label">page{{ pages|pluralize:"s" }}</td>
+                    </tr>
+                    <tr>
+                        <td class="stat">{{ users }}</td>
+                        <td class="stat-label">user{{ users|pluralize:"s" }}</td>
+                    </tr>
+                    <tr>
+                        <td class="stat">{{ projects }}</td>
+                        <td class="stat-label">project{{ projects|pluralize:"s" }}</td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+    </div>
+    <table class="u-full-width tablesorter">
+        <thead>
+            <tr>
+                <th>User</th>
+                <th>Edits</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for user in users_list %}
+            <tr>
+                <td>{{ user.username }}</td>
```

Comment:
We could link this to user pages, like:
`<td><a href="https://meta.wikimedia.org/wiki/User:{{ user.username }}>{{ user.username }}</a></td>`

On the list view we link to each entry's corresponding Wiki, but since this is an aggregate table we can just link to Meta and people can navigate from there.

Reply:
@Samwalton9 Also, On the full list of projects we could link to the corresponding project?

Reply:
Yeah that sounds good too :)